packages/core/e2e/node/claude.e2e.ts

```diff
@@ -71,12 +71,11 @@ await test("anthropic agent", async (t) => {
         },
       ],
     });
-    const { response, sources } = await agent.chat({
+    const response = await agent.chat({
       message: "What is the weather in San Francisco?",
     });
     consola.debug("response:", response.message.content);
 
-    strictEqual(sources.length, 1);
```

Comment:
@himself65 i removed `sources` (the tooloutput) 1. the name is confusing 2. it can be retrieved with events